plugin/webdevicons.vim

```diff
@@ -72,6 +72,8 @@ call s:set('g:WebDevIconsNerdTreeBeforeGlyphPadding', ' ')
 call s:set('g:WebDevIconsNerdTreeAfterGlyphPadding', ' ')
 call s:set('g:WebDevIconsNerdTreeGitPluginForceVAlign', 1)
 call s:set('g:NERDTreeUpdateOnCursorHold', 1)
+call s:set('g:WebDevIconsTabAirLineBeforeGlyphPadding', ' ')
```

Comment:
good default here because it won't change anyone's current setting since it was hardcoded as a single space before :+1: